xocto/storage/storage.py

```diff
@@ -414,6 +472,11 @@ def _build_download_filepath(self, key_path: str) -> str:
 
 
 class S3FileStore(BaseS3FileStore):
+    """
+    To customise the boto config used and/or the django settings used to populate values in the config,
+    create a subclass that overrides `_get_boto_config`.
```

Comment:
If we're allowing customisation, should the method be public instead of private? I guess this could conflict as to whether we'd want this method to be called externally.